config/vlan.py

```diff
@@ -232,3 +238,45 @@ def del_vlan_member(db, vid, port):
     except JsonPatchConflict:
         ctx.fail("{} invalid or does not exist, or {} is not a member of {}".format(vlan, port, vlan))
 
+#
+# 'static-anycast-gateway' group ('config vlan static-anycast-gateway ...')
+#
+@vlan.group(cls=clicommon.AbbreviationGroup, name='static-anycast-gateway')
+def static_anycast_gateway():
+    pass
+
+@static_anycast_gateway.command('enable')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@clicommon.pass_db
+def enable_vlan_sag(db, vid):
+    """Enable static-anycast-gatweay on VLAN interface"""
+    ctx = click.get_current_context()
+
+    log.log_info(f"'vlan static-anycast-gateway enable {vid}' executing...")
+
+    vlan = f'Vlan{vid}'
+    if not clicommon.is_valid_vlan_interface(db.cfgdb, vlan):
+        ctx.fail(f"Interface {vlan} does not exist")
+
+    db.cfgdb.mod_entry('VLAN_INTERFACE', vlan, {"static_anycast_gateway": "true"})
```

Comment:
do we need check whether static_anycast_gateway has already been configured?

Reply:
This is removed in the updated PR https://github.com/sonic-net/sonic-utilities/pull/2881